File: pkg/codegen/testing/test/testdata/simple-data-types-pp/nodejs/simple-data-types.ts
import * as pulumi from "@pulumi/pulumi";
import * as fs from "fs";

const basicStrVar = "foo";
export const strVar = basicStrVar;
export const computedStrVar = `${basicStrVar}/computed`;
export const strArrVar = [
    "fiz",
    "buss",
];
export const intVar = 42;
export const intArr = [
    1,
    2,
    3,
    4,
    5,
];
export const readme = fs.readFileSync("./Pulumi.README.md");

File: pkg/codegen/testing/test/testdata/simple-data-types-pp/dotnet/simple-data-types.cs
using System.Collections.Generic;
using System.IO;
using Pulumi;

return await Deployment.RunAsync(() => 
{
    var basicStrVar = "foo";

    return new Dictionary<string, object?>
    {
        ["strVar"] = basicStrVar,
        ["computedStrVar"] = $"{basicStrVar}/computed",
        ["strArrVar"] = new[]
        {
            "fiz",
            "buss",
        },
        ["intVar"] = 42,
        ["intArr"] = new[]
        {
            1,
            2,
            3,
            4,
            5,
        },
        ["readme"] = File.ReadAllText("./Pulumi.README.md"),
    };
});

